ksql-engine/src/test/java/io/confluent/ksql/function/UdfLoaderTest.java

```diff
@@ -167,6 +168,16 @@ public void shouldCollectMetricsWhenMetricCollectionEnabled() {
         not(nullValue()));
   }
 
+  @Test
+  public void shouldUseConfigForExtDir() {
+    final MetaStore metaStore = new MetaStoreImpl(new InternalFunctionRegistry());
+    final KsqlConfig config
+        = new KsqlConfig(Collections.singletonMap(KsqlConfig.KSQL_EXT_DIR, "src/test/resources"));
+    UdfLoader.newInstance(config, metaStore, "").load();
+    // will throw if it doesn't exist
```

Comment:
nit: also add something like "the tostring UDF function is present in a jar in the `src/test/resources` directly to enable test cases like this". 

I had to browse the code to figure out this test, and a short comment will help.